src/ItemList.js

```diff
@@ -128,58 +155,87 @@ function ItemList(props) {
+            >
+              <Add />
+            </Fab>
+          </StyledNoItems>
         </>
       )}
       {props.list && props.list.length > 0 && (
         <>
           {/* [filter-list] 1. Add text field above shopping list */}
+
+          <StyledDiv>
+            <Typography variant="h1">Your list:</Typography>
+          </StyledDiv>
+
           <label htmlFor="filter">
-            Filter Items
-            <input
+            <TextField
               id="filter"
+              variant="outlined"
               type="text"
-              placeholder="Start typing here..."
+              label="Search items"
               value={query}
               onChange={changeHandler}
             />
           </label>
+
           {/* [filter-list] 4. Add 'X' button for user to clear query field */}
           {query.length !== 0 && (
-            <button aria-label="clear field" onClick={clickHandler}>
-              X
-            </button>
+            <DeleteBtnSearch aria-label="clear field" onClick={clickHandler}>
+              <FaTrashAlt className="trash-icon" />
+            </DeleteBtnSearch>
           )}
-          <h2>Shopping List:</h2>
 
           <form>
             {Object.entries(queryObj).map(([key, value]) => {
               return (
                 value.length > 0 && (
                   <div key={key}>
-                    <h3>
-                      {key === 'week' && 'Items to buy in the next week:'}
-                      {key === 'month' && 'Items to buy in the next month:'}
-                      {key === 'longer' &&
-                        'Items to buy in the distant future:'}
-                      {key === 'inactive' && "Purchases we can't predict yet:"}
-                    </h3>
-                    <ul>
-                      {value.map((item) => (
+                    <StyledHeading>
+                      <Typography variant="h2">
+                        {key === 'week' && 'Items for next week:'}
+                        {key === 'month' && 'Items for next month:'}
+                        {key === 'longer' && 'Items for much later:'}
+                        {key === 'inactive' && 'No prediction yet:'}
+                      </Typography>
+                    </StyledHeading>
+
+                    {value.map((item) => (
+                      <Typography variant="p">
                         <Item
                           key={item.id}
                           userToken={props.userToken}
                           item={item}
                           status={key}
                         />
-                      ))}
-                    </ul>
+                      </Typography>
+                    ))}
                   </div>
                 )
               );
             })}
           </form>
+          <Fab
+            onClick={redirect}
+            type="submit"
+            color="secondary"
+            aria-label="add"
+          >
+            <Add />
+          </Fab>
```

Comment:
If there are no items, the button sticks to the bottom of the search field:

![image](https://user-images.githubusercontent.com/68360696/120079395-8df6ad80-c081-11eb-9277-04a1bfa18997.png)

You can avoid it by wrapping the button in a styled div with margin on the top.